jbmc/src/java_bytecode/java_multi_path_symex_checker.cpp

```diff
@@ -12,33 +12,24 @@ Author: Jeannie Moulton
 goto_tracet java_multi_path_symex_checkert::build_full_trace() const
 {
   goto_tracet goto_trace = multi_path_symex_checkert::build_full_trace();
-  if(options.get_bool_option("validate-trace"))
-  {
-    check_trace_assumptions(goto_trace, ns);
-    log.status() << "Trace validation successful" << messaget::eom;
-  }
+  check_trace_assumptions(
```

Comment:
Will this commit be squashed? (since it is changing code that was introduced in a prior commit in this PR)

Reply:
squashed